header/p2p/options.go

```diff
@@ -108,6 +131,7 @@ func DefaultClientParameters() ClientParameters {
 		MaxHeadersPerRequest: 64,
 		MaxAwaitingTime:      time.Hour,
 		DefaultScore:         1,
+		RequestDuration:      time.Second * 3,
```

Comment:
Should we make it slightly more relaxed, like 5 secs?

Reply:
let's make 5 sec.